book/modules/appendix1-exercises.tex

```diff
@@ -1,7 +1,85 @@
-
 \begin{exercises}
-
 	\begin{problist}
-		\prob XXX Finish
+		\prob For each equation given below, determine if it is a linear
+		equation. If not, explain what makes it nonlinear.
+		\begin{enumerate}
+			\item $\cos(4)x_{1}+\mathrm{e}y_{2}+\pi z_{3}=\mathrm{e}^{\pi}$
+
+			\item $4x_{1}+2x_{2}+5x_{4}=4x_{2}+4x_{5}+5$
+
+			\item $12x+3xy+5z=2$
+
+			\item $\cos(4)x+\sin(4)y+\tan(4)z=\sec(4)x+\csc(4)y+\cot(4)z$
+
+			\item $5x+2y+8z=\cos(y)$
+		\end{enumerate}
+
+		\prob For each system of linear equations given below: (a) write down its
+		augmented matrix, (b) use row reduction algorithm to determine if it is
+		consistent or not, (c) for each consistent system, give its complete
+		solution.
+		\begin{enumerate}
+			\item \systeme{-10x_1-4x_2+4x_3=28, 3x_1+x_2-x_3=-8, x_1+x_2-\frac{1}{2}x_3=-3}
+
+			\item \systeme{3x_1-2x_2+4x_3=54, 5x_1-3x_2+6x_3=88, x_1=-3}
+
+			\item \systeme{x_1+2x_2+4x_3-3x_4=0, 3x_1+5x_2+6x_3-4x_4=1, 4x_1+5x_2-2x_3+3x_4=3}
+
+			\item \systeme{x_1-x_2+5x_3+x_4=1, x_1+x_2-2x_3+3x_4=3, 3x_1-x_2+8x_3+x_4=5, x_1+3x_2-9x_3+7x_4=5}
+		\end{enumerate}
+
+		\prob Consider the matrix equation $M\vec x=\vec b$, where $M=\mat{2 & 8 \\ 4 & 16}$.
+		\begin{enumerate}
+			\item Show that with $\vec b=\mat{7 \\ 14}$, the system is
+				consistent.
+
+			\item Come up with another $\vec b$ to make the system consistent.
+				Justify your answer.
+
+			\item Show that with $\vec b=\mat{5 \\ 12}$， the system is
+				inconsistent.
+
+			\item Come up with another $\vec b$ to make the system inconsistent.
+				Justify your answer.
+
+			\item Prove that $M\vec x=\vec b$ is consistent if and only if $M$ and
+				$[M|\vec b]$ have the same number of pivots.
+		\end{enumerate}
+
+		\prob On Kokoro's farm, there is a cage with $35$ animals, some are chickens
+		and some are rabbits. Kokoro counted the total number of legs in the
+		cage and found that there were $94$ legs in all (notably, each chicken
+		has exactly two legs and each rabbit has four legs). Kokoro then decided
+		to use this information to figure out how many chickens and how many
+		rabbits there were.\footnote{This problem is modified based on a
+		classical Chinese problem, which first occurred in the ancient Chinese treatise
+		\emph{Mathematical Classic of Master Sun} (or \emph{Sunzi Suanjing}).}
+
+		\begin{enumerate}
+			\item Set up a system of linear equations representing this scenario.
+
+			\item Is the system consistent? If so, give the complete solution.
```

Comment:
I think "`Is the system consistent? If so, give the complete solution.`" could be moved to part (a) above. 

So far we still haven't asked the reader to find exactly how many chickens and rabbits are in the cage, so maybe part (b) could ask that?

Reply:
I was considering "give the complete solution" as "solving the problem". But, yes, it's better if we add a part to ask them to solve the problem in terms of # of chickens and rabbits. (I think it can be another subpart of part(b) though.)

Reply:
Yeah, I think subparts should work. There are a lot of interesting questions that could be asked under part (b) like `Besides the solution you found, are there any other combinations of chickens and rabbits that solves the system?` or `How do you know there can only be one possible combination of chickens and rabbits?` or `If the system were inconsistent, what would that mean about the information (total number of animals and total number of legs) Kokoro shared?`, but I'm not sure if those would be too difficult/too vague.